packages/manager/apps/telecom/src/app/telecom/telephony/line/telecom-telephony-line-main.view.html

```diff
@@ -5,15 +5,18 @@
     </div>
 
     <div class="telecom-section-header" data-ng-if="!LineCtrl.loading.init">
-        <header class="page-header" role="banner">
-            <!-- GROUP TITLE -->
-            <tuc-editable-service-name
-                data-tuc-editable-service-name-title="LineCtrl.line.getDisplayedName()"
-                data-tuc-editable-service-name-title-on-save="LineCtrl.lineNameSave"
-                data-tuc-editable-service-name-service-name="LineCtrl.line.serviceName"
-                data-maxlength="25"
-            >
-            </tuc-editable-service-name>
+        <header class="oui-page-header">
```

Comment:
I suppose component doesn't support custom component transclusion ?

Reply:
You suppose well 🙂